ElrondNetwork/elrond-go
# Update elrond-vm-common and elastic-indexer-go

## Reasoning behind the pull request
- Fixed a bug when an ESDTNFTTransfer is not parsed correctly on the transaction API and block API
  
## Proposed changes
- Updated the elrond-vm-common module
  --> PR from elrond-vm-common: https://github.com/ElrondNetwork/elrond-vm-common/pull/144
- Updated the elastic-indexer-go module
  --> PR from elastic-indexer-go: https://github.com/ElrondNetwork/elastic-indexer-go/pull/185
## Testing procedure
- Create an NFT with wrong arguments and check this log DEBUG will no longer appear.
`DEBUG[2022-11-11 14:19:09.872] [state/factory]       bech32PubkeyConverter.Encode PkBytesLength hex buff =  error = wrong size stack trace = goroutine 64770506 [running]:
runtime/debug.Stack()`

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
